academic-services/knowledge-exploration-service/tutorial-entity-linking.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Entity linking
+description: Step by step tutorial to link private paper data with MAKES paper entities
+ms.topic: tutorial
+ms.date: 10/15/2020
+---
+
+# Entity Linking
+
+This tutorial illustrates how to link private publication records with their corresponding MAKES paper entities, enabling custom entity metadata to be generated.
+
+## Prerequisites
+
+- [Microsoft Academic Knowledge Service (MAKES) subscription](get-started-setup-provisioning.md)
+- [Powershell 7](https://docs.microsoft.com/powershell/scripting/install/installing-powershell-core-on-windows?view=powershell-7&preserve-view=true)
+- [Sample private library publication records](samplePrivateLibraryData.json)
+- [Sample entity linking Powershell script](linkPrivateLibraryData.ps1)
+
+## Download samples and setup working directory
+This tutorial illustrates how a fictional library could go about linking their existing, minimal library records to the much more expansive MAG/MAKES metadata, allowing them to enhance their records and create a powerful library search website.
+
+To get started with entity linking, create a working directory on you local filesystem and download the [sample publication records](samplePrivateLibraryData.json) and the [sample entity linking script](linkPrivateLibraryData.ps1) to it.
+
+As mentioned, the sample library publication records contain minimal information:
+- Publication title
+- URL for the publication on the library's website
+
+Once the working directory has been setup, our next step is to determine the entity linking strategy.
+
+## Determine entity linking strategy
+
+Once we know what entity we want to link our data against, we can determine the appropriate linking strategy. In abstract, we'll leverage MAKES' [natural language processing](concepts-queries.md) capability via Interpret API to link entities.
```

Comment:
>Once we know what entity we want to link our data against [](start = 0, length = 57)

When was the entity we want to link against established? The way this is phrase implies that we made the determination somewhere above..